commands/template_pull.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Alex Ellis, Eric Stoekl 2017. All rights reserved.
```

Comment:
License should be:

```
// Copyright (c) Alex Ellis 2017. All rights reserved.
```
or
```
// Copyright (c) OpenFaaS Project 2017. All rights reserved.
```

Please see: https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md#license